src/utils/utilsIdentifierInput.test.ts

```diff
@@ -0,0 +1,95 @@
+import {
+  cleanIdentifierInput,
+  tagOptionalIdentifierInput,
+} from "./utilsIdentifierInput";
+
+describe("cleanIdentifierInput", () => {
+  it.each([
+    [
+      {
+        label: "phone number with whitespace",
+        value: "  +6591234567  ",
+        textInputType: "PHONE_NUMBER" as const,
+      },
+      {
+        label: "phone number with whitespace",
+        value: "+6591234567",
+        textInputType: "PHONE_NUMBER",
```

Comment:
this phone number doesnt have whitespace 😂

Reply:
i think travis meant without 😂

Reply:
hallo @dalsontws and @findtravishere, yall missed this typo and the 2 below 😂

Reply:
Sorry, my bad, i realised its before and after